### PR TITLE
Add basic GitHub profile README.md for Stoplightio organization

### PR DESCRIPTION
**Pull request by Nezteb**
Related to a conversation between Daniel, Kristen, Bailey, and myself in Slack. 🙂 

What it would look like: https://github.com/stoplightio/.github/blob/add-org-profile-readme/profile/README.md

### CODE REVIEW

**Review by daniel-white — APPROVED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<p align="center">
+  <img src="./assets/mission-team.png" width="600" height="300" />
+</p>
+
+# Social Good
+
+At Stoplight, we believe the positive impact we can make on the world isn’t limited to the API space. That’s why we work with organizations like , , and . Check out our  to learn how we offset the carbon footprint of every Stoplight employee and became a climate positive workforce.
+
+<div style="display: flex; justify-content: center;">
```
Comment (texas-kristen) [inline profile/README.md]: Not sure if these will work or if the image source has to be in the repo as well

`We offset our carbon footprint via Ecologi
We offset our carbon footprint via Ecologi`

or

`[We offset our carbon footprint via Ecologi](https://ecologi.com/stoplightinc?r=5db9f5263b387200179a1d3d)

[We offset our carbon footprint via Ecologi](https://ecologi.com/stoplightinc?r=5db9f5263b387200179a1d3d)`

**Comment by Nezteb** [comment]
@texas-kristen I came up with a solution. It's not perfect:
- It hardcodes the images. My thinking here is that if they click on the actual Ecologi link they can see an updated number. Otherwise we can just fix it when we notice that we've jumped up by another 1000 trees.
- I had to add the link as a `###` header instead of in the image itself. [Apparently the image tag itself needs an `href` with the `#gh-dark-mode-only` or else it won't work, so our previous Ecologi URL inside the image tag was preventing that from working](https://github.community/t/support-theme-context-for-images-in-light-vs-dark-mode/147981/84). Idk if we want to use a different markdown tag for the link, but it works for now.

Screen Shot 2022-03-31 at 6 35 08 PM
Screen Shot 2022-03-31 at 6 35 30 PM

**Comment by texas-kristen** [comment]
> @texas-kristen I came up with a solution. It's not perfect:
 lgtm

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# Community Impact
+
+At Stoplight, we believe the positive impact we can make on the world isn’t limited to the API space. APIs are all about connection, and at Stoplight we like to apply that connection to the community as well. Here are the community pillars we support:
+
+* **Diversity and Inclusion in STEM**
+    * Fueling the next generation of tech talents means supporting STEM-based education nonprofits and efforts to create a more equitable and inclusive tech pipeline.
+* **Environmental Impact**
+    * We work with organizations like Ecologi, Urban Roots, and OSMI. Check out our Ecologi Forest to learn how we offset the carbon footprint of every Stoplight employee and become a climate positive workforce.
+* **Open Source Contributions**
+    * We believe in benefiting the entire developer community, which is why we offer open-source tools to meet your API mocking, linting, and documentation needs. Any and all are welcome to utilize these complementary tools, add to them, and be a part of the community. [Learn more!](https://stoplight.io/open-source/)
```
Comment (texas-kristen) [inline profile/README.md]:
```suggestion
![](./assets/Sales-LinkedIn-Banner-1.png)

# About Stoplight
Stoplight is a collaborative API design platform that enables you to build excellent APIs and create scalable API programs that fuel innovation.  

# Our Core Values
At Stoplight, we ground ourselves around three guiding principles that unite us. We do our best work when we’re empowered, collaborative, and moving forward with positive intent.

*  🧘 **Practice Mindfulness** 🧘
    *  We seek to understand by listening, inquiring, and responding thoughtfully.
*  🏋️ **Be An Owner** 🏋️
    *  When we see an opportunity to make an impact, we seize it.
*  👷 **Build Together** 👷
    *  We trust in each other to add value and solve problems through teamwork.

# Mission

Stoplight’s design-first platform drives a collaborative API development experience. Design first. Develop better.
# Community Impact

At Stoplight, we believe the positive impact we can make on the world isn’t limited to the API space. APIs are all about connection, and at Stoplight, we like to apply that connection to the community as well. Here are the community pillars we support.

* **Diversity and Inclusion in STEM**
    * Fueling the next generation of tech talents means supporting STEM-based education nonprofits and efforts to create a more equitable and inclusive tech pipeline.
* **Environmental Impact**
    * We work with organizations like Ecologi, Urban Roots, and OSMI. Check out our [Ecologi Forest](https://ecologi.com/stoplightinc) to learn how we offset the carbon footprint of every Stoplight employee and become a climate-positive workforce.
* **Open Source Contributions**
    * We believe in benefiting the entire developer community, which is why we offer open-source tools to meet your API mocking, linting, and documentation needs. Any and all are welcome to utilize these complementary tools, add to them, and be a part of the community. [Learn more!](https://stoplight.io/open-source/)
```

**Review by texas-kristen — CHANGES_REQUESTED** [review]

**Review by texas-kristen — APPROVED** [review]

**Pull request merged**